doc/_templates/searchbox.html

```diff
@@ -0,0 +1,65 @@
+{#
+  Override the default searchbox from RTD theme to provide the ability to select a search method
+  (ex. built-in search, Google Custom Search, ...)
+#}
+
+{%- if ('singlehtml' not in builder) %}
```

Comment:
Not sure what `singlehtml` refers to but will the alternative search be disabled in PR builds and builds people do on their desk? The search is only going to show information from the online source which probably won't match up with these builds (not sure if it would even give local links or if the links would all be to the live documentation site)

Reply:
PR builds default to "sphinx" since they don't have a URL that starts with /latest, and same for local build on your desk. You can still switch to google (and have it saved as a preference in the browser's local storage) but to your point this will likely not be what one would want to do.

BTW `singlehtml` is for the Sphinx build mode where everything ends up in one single HTML page (something I know you're very found of ;-)), and this guard is basically the same the Read The Docs theme has in place as search is not available (needed) in single page mode.